src/main/java/org/mdbenefits/app/cli/GoogleDriveClientImpl.java

```diff
@@ -33,7 +36,11 @@ public class GoogleDriveClientImpl implements GoogleDriveClient {
     private final String GOOGLE_CREDS = System.getenv("GOOGLE_DRIVE_CREDS");
```

Comment:
So this works, but I'm debating if it should be something we pull in this way, or should be something we pull in via the appliation.yaml setup, like we do most other things.

Reply:
That seems like a question beyond the scope of this PR. I'm not particularly used to Tomcat's configuration, but I expect to see things about serving there (e.g. timeouts, file size limits, etc) rather than the application configuration for the service itself.